pkg/config/defaults.go

```diff
@@ -47,6 +47,10 @@ func setDefaults(local bool) {
 	viper.SetDefault("auth.ui_config.login_failed_message", "The credentials don't match.")
 	viper.SetDefault("auth.ui_config.login_cookie_names", "internal_auth_session")
 
+	viper.SetDefault("auth.remote_authenticator.enabled", false)
+	viper.SetDefault("auth.remote_authenticator.default_user_group", "Viewers")
+	viper.SetDefault("auth.remote_authenticator.request_timeout", 7*time.Second)
```

Comment:
I prefer the to have a default to be 10 sec or 30 sec - something smaller than 'no timeout' but something that will work for many use-cases and in case the user will want to restrict the time for authenticate, it can be done by configuration.

Reply:
Previously (In LDAP) was 7, changed to 10 👍